rl_studio/agents/f1/manual_pilot.py

```diff
@@ -6,7 +6,7 @@
 
 import gym
 
-from rl_studio.agents.f1 import settings
+from agents.f1 import settings
```

Comment:
Did you have any problems with this import? I think it reads the library dependencies better if we start from the root, don't you think?

Reply:
right, I change it today.